servers/physics/body_sw.h

```diff
@@ -223,10 +223,17 @@ class BodySW : public CollisionObjectSW {
 		angular_velocity += _inv_inertia_tensor.xform(p_j);
 	}
 
-	_FORCE_INLINE_ void apply_bias_impulse(const Vector3 &p_pos, const Vector3 &p_j) {
+
```

Comment:
A maximum amount for the length of the angular velocity change (p_max_delta_av) can be specified, negative means no limit